doc/source/docs/howto_build_docs.rst

```diff
@@ -5,7 +5,7 @@ Building the NumPy API and reference docs
 =========================================
 
 We currently use Sphinx_ for generating the API and reference
-documentation for NumPy.  You will need Sphinx 1.0.1 or newer.
+documentation for NumPy.  You will need Sphinx 3.0.0 or newer.
```

Comment:
This should be 1.8.3 or newer due to issues with the search box and our template. We pin our CI builds to 1.8.5. I think in any case 2.0.0 is the [latest releae](https://pypi.org/project/Sphinx/#history) right now. Were you thinking of another packages?

Reply:
Yes, I am changing it to 1.8.5 right away as I recall you specified this pinning to Sphinx 1.8.5 in the email yesterday. For the latest version, I did `sphinx-build --version` as suggested by an answer on stackoverflow for checking the version and it came out to be 3.0.0 so I did the same.
Sorry for the mistake.